mechanics/input_processing.py

```diff
@@ -38,7 +38,7 @@ def generate_response(npc_personality: dict, user_input: str) -> str:
         user_input, labels=["friendly", "unfriendly", "neutral"]
     )
     message_type = marvin.classify(
```

Comment:
need to add to requirements.txt yet

Reply:
https://github.com/abaddonpuff/project_G/issues/22

Reply:
Adding now :)